libs/libcommon/src/libcommon/config.py

```diff
@@ -299,6 +300,14 @@ class ProcessingGraphConfig:
                 "job_runner_version": PROCESSING_STEP_DATASET_SPLIT_NAMES_VERSION,
                 "difficulty": 20,
             },
+            "split-descriptive-stats": {
+                "input_type": "split",
+                "triggered_by": [
+                    "config-split-names-from-info",
+                    "config-split-names-from-streaming",
+                ],
+                "job_runner_version": PROCESSING_STEP_SPLIT_DESCRIPTIVE_STATISTICS_VERSION,
```

Comment:
you can add the difficulty here :)

I'd say 70 ?
```suggestion
                "job_runner_version": PROCESSING_STEP_SPLIT_DESCRIPTIVE_STATISTICS_VERSION,
                "difficulty": 70,
```